examples/Viewer.tsx

```diff
@@ -155,6 +159,24 @@ class Viewer extends React.Component<{}, ViewerState> {
         this.clearPendingFile = this.clearPendingFile.bind(this);
         this.convertFile = this.convertFile.bind(this);
         this.state = initialState;
+
+        if (props.localBackendServer) {
+            netConnectionSettings = {
+                serverIp: "0.0.0.0",
+                serverPort: 8765,
+                useOctopus: props.useOctopus,
+                secureConnection: false,
+            };
+        } else if (props.useOctopus) {
+            netConnectionSettings = {
+                serverIp: "18.223.108.15",
+                serverPort: 8765,
+                useOctopus: props.useOctopus,
+                secureConnection: false,
+            };
```

Comment:
maybe could put the other server settings here in a final else just to keep it all together in one place.